README.md

```diff
@@ -17,6 +17,18 @@ Usage
 
 In order to use _import_, require the package `ppx_import`.
 
+#### Dune and `ppx_deriving`
```

Comment:
As the section is "Using", maybe this should titled "Using with Dune"? AFAICT the instructions are not deriving-specific.